src/features/broker-protection/actions/extract.js

```diff
@@ -141,38 +146,33 @@ function findFromElement (profileElement, dataKey, extractField) {
  * @return {boolean}
  */
 function scrapedDataMatchesUserData (userData, scrapedData) {
-    if (!isSameName(scrapedData.name, userData.firstName, userData.middleName, userData.lastName)) return false
+    const matches = {
+        name: false,
+        age: false,
+        address: false,
+    };
+
+    matches.name = !isSameName(scrapedData.name, userData.firstName, userData.middleName, userData.lastName)
 
     if (scrapedData.age) {
-        if (!isSameAge(scrapedData.age, userData.age)) {
-            return false
-        }
+        matches.age = isSameAge(scrapedData.age, userData.age)
     }
 
     if (scrapedData.addressCityState) {
         // addressCityState is now being put in a list so can use matchAddressFromAddressListCityState
-        if (matchAddressFromAddressListCityState(userData.addresses, scrapedData.addressCityState)) {
-            return true
-        }
+        matches.address = !!matchAddressFromAddressListCityState(userData.addresses, scrapedData.addressCityState)
     }
 
     // it's possible to have both addressCityState and addressCityStateList
     if (scrapedData.addressCityStateList) {
-        if (matchAddressFromAddressListCityState(userData.addresses, scrapedData.addressCityStateList)) {
-            return true
-        }
+        matches.address = !!matchAddressFromAddressListCityState(userData.addresses, scrapedData.addressCityStateList)
     }
 
     if (scrapedData.addressFull) {
-        if (matchesFullAddress(userData.addresses, scrapedData.addressFull)) { return true }
-    }
-
-    if (scrapedData.phone) {
-        if (userData.phone === scrapedData.phone) { return true }
+        matches.address = !!matchesFullAddress(userData.addresses, scrapedData.addressFull)
     }
 
-    // if phone number matches
-    return false
+    return matches.name && matches.age && matches.address;
```

Comment:
this doesn't match the existing logic - previously you could match just a `name` + `address`, without age (if the scraped data didn't contain it).

regardless, we can now build upon what I started in #913 and return any amount of additional data for debug/analysis :)